.env.template

```diff
@@ -6,3 +6,6 @@ KUBERNETES_SERVICE_HOST=<host>
 KUBERNETES_SERVICE_PORT=443i
 THOTH_INFRA_NAMESPACE=<infra namespace>
 NAMESPACE=<metrics namespace>
+THOTH_GRAPH_REFRESH_SOLVER=1
+THOTH_GRAPH_REFRESH_REVSOLVER=1
+THOTH_GRAPH_REFRESH_SECURITY=1
```

Comment:
can you add `THOTH_GRAPH_REFRESH_COUNT` which is missing and explain the env variable in the README?

Reply:
Thanks